internal/engine/eval/vulncheck/review.go

```diff
@@ -312,30 +275,64 @@ func (ra *reviewPrHandler) findPreviousReview(ctx context.Context) error {
 	}
 
 	ra.minderReview = nil
-	for _, r := range reviews {
-		if strings.HasPrefix(r.GetBody(), reviewBodyMagicComment) && r.GetState() != "DISMISSED" {
-			ra.minderReview = r
+	for i := len(reviews) - 1; i >= 0; i-- {
```

Comment:
This is done because one of the calls returns reviews in reverse order, right? Could you add a comment here so that we remember the reason later?

Reply:
Yes exactly. Very fair point. Will add a comment.